### ISSUE TITLE
love.event.wait returning nil and mouse move events

### ISSUE THREAD

**Issue by slime73** [love2d/love#79]
**[Original report](https://bitbucket.org/rude/love/issue/74) by Anonymous.**

----------------------------------------

I considered using love.event.wait for a certain little game I was making that is entirely input driven. It turned out it returned a nil in place of the event return value every time I moved my mouse. I don't know if the usual love.event.poll loop ever does this, as the nil the iterator returns should be interpreted as ending the for loop. (You may want to check what happens internally.)

As it turns out, I could use this feature. I could write a wrapper around love.event.wait that turned these nil values into mouse move events. I don't use love.event.wait very often, though.

I would like the devs to clarify this. Is this behavior of returning nil values the same on all platforms? Is it intended to be there, or are you planning to remove it in the future? Are you planning to introduce support for mouse move events into Löve itself (You can write your own mouse tracker very easily. Thus it's not necessary.)

Basically, I don't know if this is a bug, intended feature and/or completely irrelevant. I propose that you tell me!
